internal/vault/vault_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	clientcorev1 "k8s.io/client-go/listers/core/v1"
```

Comment:
Move to block below, as per https://cert-manager.io/docs/contributing/coding-conventions/#organizing-imports